src/sentry_webhook/post_sentry_event_to_salesforce.ts

```diff
@@ -151,6 +151,33 @@ function getSalesforceFormData(
   return form.join('&');
 }
 
+// Returns a picklist value that is allowed by SalesForce for the OS record.
+function toOSPicklistValue(value: string | undefined): string | undefined {
+  if (!value) {
+    console.warn('No OS found');
+    return undefined;
+  }
+
+  const normalizedValue = value.toLowerCase();
```

Comment:
I thought about both of those, but there seem to be a number of variants (e.g. Arch Linux, Ubuntu Linux, Fedora, macOS, Mac OS X, etc.). This is based on which SDK is used for the event, and even then there is no exhaustive list and I believe is parsed from the User Agent or something like the value of `sw_vers --ProductName`. It seems a little less brittle to be more forgiving.

Having said that, I wonder if Linux should just be the fallback since there could be many other distros and we don't support other platforms anyway.